front-end/src/components/tab/TabWrapper.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="tabs">
+    <ul class="tabs__header">
+      <li v-for="title in tabTitles" :key="title" @click="selectedTab = title">
+        {{ title }}
+      </li>
+    </ul>
+    <slot />
+  </div>
+</template>
+
+<script>
```

Comment:
[Should]
script setup

Reply:
Đoạn này em không biết chuyển về script setup kiểu gì luôn anh ơi